js/packages/cli/src/commands/updateFromCache.ts

```diff
@@ -62,6 +62,15 @@ export async function updateMetadataFromCache(
   cacheContent: any,
   newCacheContent: any,
 ) {
+  const utf8Encode = new TextEncoder();
```

Comment:
i think even more so could `deriveCandyMachineV2ProgramAddress` be used in this case @samuelvanderwaal